dotnet/build-packages.sh

```diff
@@ -1,8 +1,8 @@
 
-RELEASE_VERSION="2.8.000"
-DEVELOPMENT_VERSION="0.2.111-prerelease"
-AGENT_DOWNLOAD_URL="http://s3.amazonaws.com/dsd6-staging/windows/agent7/buildpack/agent-binaries-7.35.1-1-x86_64.zip"
-TRACER_DOWNLOAD_URL="https://github.com/DataDog/dd-trace-dotnet/releases/download/v2.8.0/windows-tracer-home.zip"
+RELEASE_VERSION="2.9.000"
+DEVELOPMENT_VERSION="0.2.112-prerelease"
+AGENT_DOWNLOAD_URL="http://s3.amazonaws.com/dsd6-staging/windows/agent7/buildpack/agent-binaries-7.35.2-1-x86_64.zip"
```

Comment:
I thought we disabled this bump? 🤔 
```suggestion
AGENT_DOWNLOAD_URL="http://s3.amazonaws.com/dsd6-staging/windows/agent7/buildpack/agent-binaries-7.35.1-1-x86_64.zip"
```

Reply:
No I decided it was bad to sneak it in my last PR. I'll push that today with a few other changes. 
And meanwhile I'm testing with the new tracer for now :)

Reply:
I just noticed you snuck the new agent into the _last_ release. Sneaky devil

Reply:
Yep :) You said, that you don't want to do it, I never said I didn't :) 
To be fair I had already deployed it with the new agent. Ant the perfs were ok. So I pushed it as is. Also why I've tested today with it.